azure_jumpstart_arcbox/artifacts/Bootstrap.ps1

```diff
@@ -321,21 +335,21 @@ Write-Header "Configuring Logon Scripts"
 if ($flavor -eq "Full" -Or $flavor -eq "ITPro") {
     # Creating scheduled task for ArcServersLogonScript.ps1
     $Trigger = New-ScheduledTaskTrigger -AtLogOn
-    $Action = New-ScheduledTaskAction -Execute "PowerShell.exe" -Argument $Env:ArcBoxDir\ArcServersLogonScript.ps1
+    $Action = New-ScheduledTaskAction -Execute "pwsh.exe" -Argument $Env:ArcBoxDir\ArcServersLogonScript.ps1
```

Comment:
Maybe define pwsh variable and use it instead of pwsh.exe repeateadly

Reply:
Good idea, I have now added a variable referencing the executable.